app/templates/views/support/index.html

```diff
@@ -24,7 +24,7 @@
     {{ page_footer('Continue') }}
   {% endcall %}
 
-  <p class="govuk-body">If you have a .gov.uk, .gov.scot or .gov.wales email address, you can also <a class="govuk-link govuk-link--no-visited-state" href="https://ukgovernmentdigital.slack.com/messages/C0E1ADVPC">contact us on Slack</a>.</p>
+  <p class="govuk-body">If you have a .gov.uk, .gov.scot, .gov.wales, .police.uk or .mod.uk email address, you can also <a class="govuk-link govuk-link--no-visited-state" href="https://ukgovernmentdigital.slack.com/messages/C0E1ADVPC">contact us on Slack</a>.</p>
```

Comment:
The list is looking a bit long now. I’m reluctant to turn it into a bullet point list as I think that makes it too prominent.

We could do this instead, if we think .gov.uk .gov.scot and .gov.wales users will understand what we mean.

```suggestion
  <p class="govuk-body">If you have a .gov, .police.uk or .mod.uk email address, you can also <a class="govuk-link govuk-link--no-visited-state" href="https://ukgovernmentdigital.slack.com/messages/C0E1ADVPC">contact us on Slack</a>.</p>
```

Reply:
I'm not sure about this, .gov is an actual domain, and it's for US government 😓